net/pfSense-pkg-freeradius2/files/usr/local/pkg/freeradius.inc

```diff
@@ -447,15 +463,16 @@ EOD;
 	conf_mount_rw();
 	file_put_contents(FREERADIUS_ETC . '/raddb/radiusd.conf', $conf);
 	conf_mount_ro();
-	
+
 	// freeradius_sqlconf_resync() is pointing to this function because we need
 	// to run freeradius_serverdefault_resync() and after that restart freeradius.
 	freeradius_plainmacauth_resync();
 	freeradius_motp_resync();
 	freeradius_serverdefault_resync();
-	
+
 	// This is to fix the mysqlclient.so which gets lost after reboot
-	exec("ldconfig -m /usr/local/lib/mysql");
+	// XXX: Probably some PBI leftover, remove?
+	mwexec("/sbin/ldconfig -m /usr/local/lib/mysql");
```

Comment:
I strongly believe this could be removed. MySQL package registers itself to run when /etc/rc.d/ldconfig is called by rc script